cextern/expat/Makefile

```diff
@@ -0,0 +1,1027 @@
+# Makefile.in generated by automake 1.15.1 from Makefile.am.
```

Comment:
I'm not sure it makes sense to include the Makefile since this is generated from the configure script?